apps/condo/domains/organization/components/EmployeeProfileForm/EmployeeRoleSelect.tsx

```diff
@@ -15,18 +16,20 @@ export const EmployeeRoleSelect: React.FC<IEmployeeRoleSelectProps> = (props) =>
     const { objs: employeeRoles, loading, error } = OrganizationEmployeeRole.useObjects({ where: { organization: { id: employeeOrganizationId } } })
     const options = useMemo(() => employeeRoles.map((employeeRole) => {
         const convertedOption = OrganizationEmployeeRole.convertGQLItemToFormSelectState(employeeRole)
-
+        console.log('convertedOption', convertedOption)
+        console.log('hii')
```

Comment:
Please remove console.logs

Reply:
Please add WIP. It’s looks like not ready for review